invenio_communities/assets/semantic-ui/js/invenio_communities/members/invitations/RequestStatus.js

```diff
@@ -0,0 +1,51 @@
+// This file is part of InvenioRequests
+// Copyright (C) 2022 CERN.
+//
+// Invenio RDM Records is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+import React from "react";
+import RequestStatusIcon from "@js/invenio_requests/request/RequestStatusIcon";
+import { i18next } from "@translations/invenio_communities/i18next";
+
+export const SubmitStatus = () => (
+  <div>
+    <RequestStatusIcon status="submitted" />
```

Comment:
status should not be hardcoded in the prop, but it should be passed from the parent component